build.gradle

```diff
@@ -452,6 +452,12 @@ task yarn(dependsOn: mapIntermediaryJar, type: JavaExec) {
 	jvmArgs "-Xmx2048m"
 }
 
+task yarnUnpicked(dependsOn: "yarn") {
```

Comment:
I don't see why we should keep this. The yarn commands are always user-oriented and we would not expect a bot to call this task fully automated.

Also remember to update readme